### PR TITLE
Proof reading Readme, Step0, Step1

### PR DESCRIPTION
**Pull request by diraulo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,33 @@
 ### Learn To Code
 
-Welcome to the Learn To Code Workshop. During the 4 days we are going to spend together we will explore some of the techniques professional developers are using
-in their development process. The idea with this workshop is to showcase some of those techniques and put them in to practice.
+Welcome to the Learn To Code Workshop. For the upcoming 4 days we'll spend together, will explore some techniques used by professional developers during their development process. The idea with this workshop is to showcase some of those techniques and put them in to practice.
```
Comment (tochman) [inline README.md]: Hmm.. not 
```
For the upcoming 4 days we'll spend together, ** we ** will explore some...
```?

**Comment by tochman** [comment]
Looks good @diraulo. Had one small comment only. Please review and I'll merge it in. :-)

File: README.md (hunk 1 above)
Comment (diraulo) [inline README.md]: okay :wink:

**Comment by diraulo** [comment]
@tochman it's done

**Pull request merged**